### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sungyonhong**
updating the intro

### CODE REVIEW

**Comment by sungyonhong** [comment]
updating intro

**Comment by sungyonhong** [comment]
updating intro

**Comment by sungyonhong** [comment]
updating intro

File: examples/SettingUpAMLComputeWorkspaces/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 [![Deploy to Azure](http://azuredeploy.net/deploybutton.png)](https://deploy.azure.com/?repository=https://github.com/BlueGranite/dogbreeds/tree/colby#/form/setup)
 
-The purpose of this tutorial is to guide administrators on creating and preparing Azure Machine Learning (AzureML) service workspaces for data scientists. AzureML empowers teams to collaboratively build, train, and deploy machine learning models.
+Enterprise administrators are responsible for managing Azure Machine Learning Service (AMLS) compute quota and compute clusters. This provide guidance discusses how compute resources can be shared in AMLS, and how workspaces and compute clusters can be set up to help with resource management. Assigning right level of roles to users is an importnat task for administrators.
```
Comment (jpe316) [inline examples/SettingUpAMLComputeWorkspaces/README.md:5]: Multiple typos and grammatical errors. Please fix.

**Review by jpe316 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**